<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>damagecalculatorwindow</class>
 <widget class="QMainWindow" name="damagecalculatorwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>688</width>
    <height>574</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Damage Calculator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="fortifyBox">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>260</y>
      <width>71</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Fortify</string>
    </property>
    <widget class="QRadioButton" name="faint1RadioButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>117</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Cartx1</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="faint2RadioButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>117</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Cartx2</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="fortifyNoneRadioButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>117</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>None</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="weaponTypeLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Weapon Type</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="dangerBox">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>260</y>
      <width>120</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Danger</string>
    </property>
    <widget class="QRadioButton" name="adrenaline2RadioButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>117</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Adrenaline+2</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="heroicsRadioButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>117</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Heroics</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="dangerNoneRadioButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>117</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>None</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="seedPillBox">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>170</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Might Item</string>
    </property>
    <widget class="QRadioButton" name="mightSeedRadioButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>117</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Seed</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="mightPillRadioButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Pill</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="mightNoneRadioButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>117</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>None</string>
     </property>
    </widget>
   </widget>
   <widget class="QListWidget" name="weapon_list_widget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>261</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="AuXBox">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>0</y>
      <width>71</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>AuX</string>
    </property>
    <widget class="QRadioButton" name="AuSRadioButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>AuS</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="AuMRadioButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>AuM</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="AuLRadioButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>AuL</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="AuXLRadioButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>80</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>AuXL</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="AuXNoneRadioButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>100</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>None</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="otherBox">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>330</y>
      <width>151</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>Other</string>
    </property>
    <widget class="QCheckBox" name="sharpnessCheckBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Sharpness+1</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="weaknessExploitCheckBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>151</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Weakness Exploit</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="powerCharmCheckBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>80</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Power Charm</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="criticalDrawCheckBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Critical Draw</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="powerTalonCheckBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>100</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Power Talon</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="rawHoneCheckBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>120</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Raw Hone</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="criticalElementCheckBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>140</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Critical Element</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="wyvernStoneRawCheckBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>160</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Wyvern (Raw)</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="wyvernStoneEleCheckBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>180</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Wyvern (Ele)</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="kitchenBox">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>170</y>
      <width>120</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Kitchen</string>
    </property>
    <widget class="QRadioButton" name="AuSKitchenRadioButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>117</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Attack Up S</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="AuLKitchenRadioButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>117</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Attack Up L</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="kitchenNoneRadioButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>117</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>None</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gloveBox">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>0</y>
      <width>151</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Glove</string>
    </property>
    <widget class="QRadioButton" name="latentPower1RadioButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Latent Power+1</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="latentPower2RadioButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Latent Power+2</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="challenger1RadioButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Challenger+1</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="challenger2RadioButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>80</y>
       <width>117</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Challenger+2</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="gloveNoneRadioButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>120</y>
       <width>117</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>None</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="peakPerformanceRadioButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>100</y>
       <width>161</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Peak Performance</string>
     </property>
    </widget>
   </widget>
   <widget class="QListWidget" name="motionValueListWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>261</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="motionValueStringLabel">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>350</y>
      <width>181</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>[0]</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="huntingHornBox">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>170</y>
      <width>120</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Hunting Horn</string>
    </property>
    <widget class="QRadioButton" name="attackUpSRadioButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>117</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Attack Up (S)</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="attackUpLRadioButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>117</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Attack Up (L)</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="replayCheckBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>100</y>
       <width>97</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Replay</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="HHNoneRadioButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>120</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>None</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="affinityUpCheckBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Affinity Up</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="elementUpCheckBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>80</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Element Up</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="caclulatePushButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>370</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculate</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="criticalEyeBox">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>0</y>
      <width>120</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Critical Eye</string>
    </property>
    <widget class="QRadioButton" name="criticalEye1RadioButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>117</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Critical Eye+1</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="criticalEye2RadioButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>117</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Critical Eye+2</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="criticalEye3RadioButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>117</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Critical Eye+3</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="criticalGodRadioButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>80</y>
       <width>117</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Critical God</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="CENoneRadioButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>100</y>
       <width>117</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>None</string>
     </property>
    </widget>
   </widget>
   <widget class="QListWidget" name="weapon_type_list_widget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>131</width>
      <height>141</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>ChargeBlades</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>InsectGlaives</string>
     </property>
    </item>
   </widget>
   <widget class="QGroupBox" name="elementBox">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>0</y>
      <width>131</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Element</string>
    </property>
    <widget class="QRadioButton" name="element1RadioButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>117</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Element+1</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="element2RadioButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>117</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Element+2</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="element3RadioButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>117</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Element+3</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="elementAtkUpCheckBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>80</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>ElementAtkUp</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="elementNoneRadioButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>100</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>None</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="weaponLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Weapon</string>
    </property>
   </widget>
   <widget class="QLabel" name="motionValueLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Motion Value</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="damageTabWidget">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>400</y>
      <width>221</width>
      <height>121</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="displayBasicTab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QLabel" name="totalDamageLabel">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>94</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Total Damage</string>
      </property>
     </widget>
     <widget class="QLabel" name="totalDamageFloatLabel">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>0</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>0.00</string>
      </property>
     </widget>
     <widget class="QLabel" name="rawDamageLabel">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>89</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Raw Damage</string>
      </property>
     </widget>
     <widget class="QLabel" name="rawDamageFloatLabel">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>30</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>0.00</string>
      </property>
     </widget>
     <widget class="QLabel" name="elementalDamageLabel">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>60</y>
        <width>81</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Ele Damage</string>
      </property>
     </widget>
     <widget class="QLabel" name="eleDamageFloatLabel">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>60</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>0.00</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="displayAdvancedTab">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>688</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
